build/rpmfc.c

```diff
@@ -1181,6 +1181,9 @@ static const struct skipped_extension_s skipped_extensions[] = {
 	{ ".h",  "C Header",                 "text/x-c"   },
 	{ ".la", "libtool library file",     "text/plain" },
 	{ ".pc", "pkgconfig file",           "text/plain" },
+	{ ".html", "HTML document, Unicode text, UTF-8 text", "text/html" },
```

Comment:
I don't think we should claim it to be Unicode or UTF-8 if we don't even look at it.